src/charm.py

```diff
@@ -28,7 +30,7 @@ class MySQLCharm(CharmBase):
 
     def __init__(self, *args):
         super().__init__(*args)
-        self._stored.set_default(mysql_setup={})
+        self._stored.set_default(mysql_setup={"MYSQL_ROOT_PASSWORD": False})
         self._stored.set_default(mysql_initialized=False)
         self.image = OCIImageResource(self, "mysql-image")
```

Comment:
I don't understand this. It does not seem to be used anywhere.

Reply:
It is used in `mysql_root_password` property